datawig/utils.py

```diff
@@ -45,6 +46,31 @@
 logger.setLevel("INFO")
 
 
+def merge_two_dicts(x, y):
```

Comment:
Why is this method required when we already have similar ? https://github.com/awslabs/datawig/blob/master/datawig/utils.py#L60

Reply:
It isn't. Will fix in the next commit.